authors/apps/articles/views.py

```diff
@@ -72,7 +72,7 @@ def create(self, request):
         serializer_data = request.data.get('article', {})
 
         time_to_read = Article.article_reading_time(serializer_data['body'])
-        serializer_data['readingTime'] = time_to_read
+        serializer_data['reading-time'] = time_to_read
```

Comment:
Good job fixing this @mendozabree.
This should be an underscore, not a dash.
Please check to make sure.

Reply:
Checked it out, thanks @RonKbS it is actually supposed to be an `underscore` let me change that.